backstop/tests/skip-to-content.html

```diff
@@ -0,0 +1,196 @@
+<!--
+  Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+  See LICENSE.md in the project root for license terms and full copyright notice.
+-->
+<!DOCTYPE html>
+<html
+  lang="en-US"
+  id="theme"
+>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1"
+    >
+    <title>skip-to-content | iTwinUI</title>
+    <link
+      rel="stylesheet"
+      href="../../lib/css/all.css"
+    />
+    <link
+      rel="stylesheet"
+      href="../assets/demo.css"
+    />
+    <style>
+      section {
+        width: auto !important;
+      }
+    </style>
+    <script
+      type="module"
+      src="https://cdn.skypack.dev/@itwin/itwinui-icons-elements/imodel-hollow"
+    ></script>
+    <script
+      type="module"
+      src="https://cdn.skypack.dev/@itwin/itwinui-icons-elements/caret-down-small"
+    ></script>
+    <script
+      type="module"
+      src="https://cdn.skypack.dev/@itwin/itwinui-icons-elements/chevron-right"
+    ></script>
+    <script
+      type="module"
+      src="https://cdn.skypack.dev/@itwin/itwinui-icons-elements/notification"
+    ></script>
+    <script
+      type="module"
+      src="https://cdn.skypack.dev/@itwin/itwinui-icons-elements/help-circular-hollow"
+    ></script>
+    <script
+      type="module"
+      src="https://cdn.skypack.dev/@itwin/itwinui-icons-elements/more-vertical"
+    ></script>
+    <script src="../assets/theme.js"></script>
+    <script src="../assets/userIcon.js"></script>
+  </head>
+  <body class="iui-body">
+    <a
+      href="#demo-skip-to-here"
+      class="iui-skip-to-content"
+      id="test-1"
+    >Skip to main content</a>
+    <theme-button></theme-button>
+    <h1>Skip to content</h1>
+    <hr />
+
+    <section id="demo-other-content">
```

Comment:
I havent read too much about `skip-to-content` but isnt the purpose of it to scroll through not important content to be able to see important/main one in viewport? If so, could we make "not important" content much bigger? I find current html a bit weird, cause the "important" content is literally there.